btree/btree.c

```diff
@@ -3032,6 +3087,8 @@ static int64_t btree_put_root_split_handle(struct m0_btree_op *bop,
 	node_done(&node_slot, bop->bo_tx, true);
 	node_seq_cnt_update(lev->l_node);
 	node_fix(lev->l_node, bop->bo_tx);
+	btree_callback_add(oi->i_callback, lev->l_node);
```

Comment:
you can avoid  calling btree_callback_add(oi->i_callback, lev->l_node); as it will get already added in tree_put_makespace_phase at line no. 3224 or 3300

Reply:
ok.